blockchain/README.md

```diff
@@ -8,15 +8,15 @@
 * 3. Copy Address of NftTheWorld
 * 4. in TokenFactory execute function addtoAdminContracts() with just copied address
 * Optional: In NftTheWorld: with addtoAdmins() add your teammates to contract, so they are elligible to test as well
-* Feel free to Test!
+* Feel free to test!
 
 
 ## Verifying & Publishing 
 * 1. After deploying contract(s) go to https://kovan.etherscan.io/ and look for contract address
 * 2. Hit tab "Contract" mid-page and just below click "Verify & Publish"
-* 3. Select: Solidity (Single file), Compiler Version v0.8.0 & respective license type & continue
-* 4. In next step, copy & paste contract & source code, and eventual Constructor Arguments & other setting (not needed atm)
-* 5. Hopefully find successfully generated ByteCode on next page
+* 3. Select: Solidity (Single file), Compiler Version v0.8.10 & respective license type & continue
```

Comment:
But we are using 8.0 aren't we?
As it say: pragma solidity ^0.8.0; and also for verification I've always used 8.0.

Reply:
^0.8.0 means >=0.8.0 and I always use 0.8.10 when verifying :D But can also change if you'd like me to :)

Reply:
I would leave it like that, since minor changes are mostly security fixes and won't break anything

Reply:
Ahh okay just thought that during verification it would still lead to an error if the minimal-versions were not matching, but if  thats not the case this is fine aswell